eva-accession-import-dbsnp2/src/main/resources/application.properties

```diff
@@ -0,0 +1,21 @@
+spring.batch.job.names=IMPORT_DBSNP_JSON_VARIANTS_JOB
+
+# dbSNP JSON input source
+parameters.input=src/test/resources/input-files/chrY50.json.bz2
+# reference sequence accession
+parameters.assembly=
+
+parameters.chunkSize=1000
+parameters.forceRestart=false
+
+# MongoDB for storing imported accessions
+spring.data.mongodb.host=localhost
+spring.data.mongodb.port=27017
+spring.data.mongodb.authentication-database=admin
+spring.data.mongodb.username=appAdmin
+spring.data.mongodb.password=evapipeline
+
+mongodb.read-preference=primaryPreferred
+
+# random port (randomizes default 8080)
+server.port=0
```

Comment:
Any reason to set this parameter?

Reply:
This allows to setup a default Data source database at a random port instead of default 8000. I would get failures because this port hadn't been closed by earlier job exec. This can be removed. This is that data source required by Job executor to track job progress

Reply:
This should not be necessary. This issue may be due to the auto-configuration being disabled.

Reply:
Agreed, doesn't make sense to keep this here.